src/rmq/Config.cmake.in

```diff
@@ -0,0 +1,11 @@
+@PACKAGE_INIT@
+
+if(NOT TARGET rmqcpp)
+    foreach(dep bal;OpenSSL;ZLIB)
+        find_package(${dep})
+    endforeach()
```

Comment:
These are needed so that the consuming upstream application does not need to explicitly do `find_package(dep)` for deps of rmqcpp.